### PR TITLE
Language added as an optional variable to get_tts

### PR DESCRIPTION
**Pull request by Joanguitar**
I'm working on adding multi-language support to [HiveMind-core] and this step is essential for making it work with OVOS-plugin-manager.
I will be creating PR for many of your repositories if you don't mind.

### CODE REVIEW

File: ovos_tts_plugin_responsivevoice/__init__.py [hunk 1]
```diff
@@ -49,7 +49,13 @@ def get_tts(self, sentence, wav_file):
         Returns:
             Tuple ((str) written file, None)
         """
-        self.engine.get_mp3(sentence, wav_file)
+        if lang and lang == self.lang:
+            self.engine.get_mp3(sentence, wav_file)
+        else:
+            gender = self.config.get("gender", "male")
```
Comment (JarbasAl) [inline ovos_tts_plugin_responsivevoice/__init__.py]: make this self.gender in `__init__`

File: ovos_tts_plugin_responsivevoice/__init__.py [hunk 2]
```diff
@@ -21,15 +21,15 @@ class ResponsiveVoiceTTSPlugin(TTS):
     """Interface to ResponsiveVoice TTS."""
 
     def __init__(self, lang="en-us", config=None):
-        config = config or {"lang": lang,
+        self.config = config or {"lang": lang,
```
Comment (JarbasAl) [inline ovos_tts_plugin_responsivevoice/__init__.py]: `self.config` is assigned in the base class, dont assign to `self` here, but leave the `self.config` after the super call in pitch rate and volume

File: ovos_tts_plugin_responsivevoice/__init__.py [hunk 3]
```diff
@@ -49,7 +49,13 @@ def get_tts(self, sentence, wav_file):
         Returns:
             Tuple ((str) written file, None)
         """
-        self.engine.get_mp3(sentence, wav_file)
+        if lang and lang == self.lang:
```
Comment (JarbasAl) [inline ovos_tts_plugin_responsivevoice/__init__.py]: should be `if not lang or lang == self.lang`, otherwise the else will be called when `lang==None`

Comment (Joanguitar) [reply ovos_tts_plugin_responsivevoice/__init__.py]: Sorry, I realized this while adding the lang option to the other repos, already corrected

**Comment by Joanguitar** [comment]
I will correct the self.config thing and squash the commits into a single one

**Comment by Joanguitar** [comment]
Everything should be good now

File: ovos_tts_plugin_responsivevoice/__init__.py (hunk 1 above)
Comment (Joanguitar) [reply ovos_tts_plugin_responsivevoice/__init__.py]: Done, squashed and pushed.

**Pull request merged**